pandas/core/groupby/generic.py

```diff
@@ -758,7 +758,7 @@ def aggregate(self, func_or_funcs, *args, **kwargs):
         if isinstance(func_or_funcs, compat.string_types):
             return getattr(self, func_or_funcs)(*args, **kwargs)
 
-        if isinstance(func_or_funcs, collections.Iterable):
+        if isinstance(func_or_funcs, compat.Iterable):
```

Comment:
in theory we could add a lint rule to avoid using ``collection.Iterable``, and instead just use our compat

Reply:
I think we're close enough to py3-only that we'll be OK. Writing that regex would be a bit fiddly since we'd need to enumerate all the ABCs.

Reply:
true, certainly a follow up issue is ok